rules-reviewed/openshift/session.windup.xml

```diff
@@ -25,63 +25,77 @@
               </xmlfile>
             </when>
             <perform>
-                <hint title="HTTP Session replication" effort="5" category-id="openshift-mandatory">
+                <hint title="HTTP Session replication" effort="3" category-id="cloud-mandatory">
                     <message>
                     <![CDATA[
-                       The `web.xml` deployment descriptor for a Web application archive contains the `<distributable>` subelement.  
-                       To configure EAP clustering in OpenShift, please follow the instruction in the following links:
+                       Session replication ensures that client sessions are not disrupted by failovers of nodes in a cluster.  
+                       Each node in the cluster shares information about ongoing sessions and can take over sessions if a node disappears.  
+                       To enable clustering of a web application, you must configure the application to be distributable [1][6]:  
```

Comment:
As this rule detect the "distributable" tag, there is no need to explain how to make an application distributable. I would recommend reformulating and telling that the distributable tag enables the clustering of HTTP sessions.